src/python/WMCore/WMSpec/Steps/Diagnostics/AlcaHarvest.py

```diff
@@ -41,7 +41,7 @@ def __call__(self, errCode, executor, **args):
         if not os.path.exists(jobRepXml):
             # no report => Error
             msg = "No Job Report Found: %s" % jobRepXml
-            executor.report.addError(50115, "MissingJobReport", msg)
+            executor.report.addError(executor.step._internal_name, 50115, "MissingJobReport", msg)
```

Comment:
Oh really, it was buggy since a long time because we were passing the exit code as stepName? :)

Reply:
Wow, really? that was my first reaction too. :-)